_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible, for it to be finalized in the cEP?"
```

Comment:
I too think that line is not needed here.

Reply:
I'd suggest that you split the question into two. 
Q1: ```What is a cEP?```
Q2: ```What is it important to do a pre-GSoC analysis and design a project proposal?```
Since in the answer you are specifying that the project proposal should be made as early as possible, that part could be omitted from the question.

Reply:
I think you mistyped Q2, it should be `Why is it ...` am i right ?

Reply:
Will split the question into two parts, and commit them soon 😇

Reply:
can you please check it, if it is looking correct or not ?

Reply:
@frextrite please review latest commit